spec/config_spec.rb

```diff
@@ -219,6 +219,11 @@
   end
 
   describe "#[]" do
+    it "returns a BasicObject" do
```

Comment:
This makes it sound like it always returns BasicObject. It is missing `context`. That said, I'd rather remove this altogether.

Reply:
please see https://github.com/pry/pry/pull/1890#discussion_r234459324 - i don't think we should intentionally not support storing BasicObject as a value just because there is a style preference towards `is_a?`.

Reply:
Could tweak the description? It is still a bit confusing because it sounds like it always returns a BasicObject.